controller/workitem.go

```diff
@@ -449,17 +464,24 @@ type WorkItemConvertFunc func(*http.Request, *workitem.WorkItem, *app.WorkItem)
 
 // ConvertWorkItems is responsible for converting given []WorkItem model object into a
 // response resource object by jsonapi.org specifications
-func ConvertWorkItems(request *http.Request, wis []workitem.WorkItem, additional ...WorkItemConvertFunc) []*app.WorkItem {
+func ConvertWorkItems(request *http.Request, wits []workitem.WorkItemType, wis []workitem.WorkItem, additional ...WorkItemConvertFunc) []*app.WorkItem {
 	ops := []*app.WorkItem{}
-	for _, wi := range wis {
-		ops = append(ops, ConvertWorkItem(request, wi, additional...))
+
+	// take the smallest length as the loop length
```

Comment:
Why this safety net? Can you add a comment when this occur?

Reply:
@baijum I just want to be sure that one does not call this function with more WIs that WITs or vice versa.

Reply:
It might make sense to error if this happens and that requires the conversion function to return an error. Will look into that.

Reply:
An error makes sense when length doesn't match.

Reply:
I've changed the function signature of `ConvertWorkItem(s)` to also return an `error`: f7d688fe025210e5b9d2c51c30802e1a36a93258